### PR TITLE
removed using namespace std

### PR DESCRIPTION
**Pull request by iamrajiv**
#### Description of Change

Removed `using namespace std;`

#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#New-File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTION.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] Sort by alphabetical order
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by kvedala** [comment]
please keep the indentations to 4 spaces

File: math/sieve_of_eratosthenes.cpp [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
  */
 
 #include <iostream>
-using namespace std;
 
 #define MAX 10000000
 
 int isprime[MAX];
 
 /*
```
Comment (ayaankhan98) [inline math/sieve_of_eratosthenes.cpp:15]: please charge comment style
it should be like 
```
/**
 *
 */
```

File: math/sieve_of_eratosthenes.cpp [hunk 2]
```diff
@@ -1,32 +1,27 @@
...
+ * Time Complexity: O(N * log N)
+ * Space Complexity: O(N)
+**/
 
 #include <iostream>
-using namespace std;
 
 #define MAX 10000000
 
 int isprime[MAX];
```
Comment (ayaankhan98) [inline math/sieve_of_eratosthenes.cpp:13]: instead of defining this much large array, use dynamic memory allocation to do the same task

Comment (iamrajiv) [reply math/sieve_of_eratosthenes.cpp:13]: @ayaankhan98 I have implemented this algorithm in #821 in a much-optimized way because of this #820

**Comment by ayaankhan98** [comment]
moved to #821

**Pull request closed without merging**